tests/test_selection/test_single_feature_performance.py

```diff
@@ -136,7 +134,7 @@ def test_raises_warning_if_no_feature_selected(load_diabetes_dataset):
         threshold=10,
     )
     with pytest.warns(UserWarning):
-        warnings.warn(sel.fit(X, y), UserWarning)
+        sel.fit(X, y)
```

Comment:
thanks for this fix, it was helpful for another PR that I am working on. I will merge that PR with this solution today, so I think we can remove this fix from this PR :)